File: DemoTest/Models/KO_DemoModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace DemoTest.Models
{
    public class KO_DemoModel : KO_BaseModel
    {
        [CRUDPresentation(Label="Thats the Prename")]
        public string Prename { get; set; }
        [CRUDPresentation()]
        public string Surname { get; set; }
        public string Birthday { get; set; }

        [ModelMethodDescriber(koComputed = false, Description = "Handling submit event", DisplayMode=ModelMethodDisplayMode.DisplayAsSubmit)]
        [CRUDPresentation(Label = "Save now")]
        public void Submit()
        {
        }
    }
}

File: KO_ModelBuilder/KO_DemoModel.js
var KO_DemoModel = function (context) {
	var self = this;
	self.koContext = null;

	// Generated Modelobservables
	self.Prename = ko.observable();
	self.Surname = ko.observable();
	self.Birthday = ko.observable();
	// END

	// Constructor
	self.init = function (context) {

	};

	// Generated ModelMethods
	// Handling submit event
    self.Submit = function() {
		// CODE HERE
	};
	// END

	self.init();
}
